examples/README.md

```diff
@@ -520,6 +520,12 @@ python3 -m http.server --directory examples/wasm
 ruby -run -ehttpd examples/wasm
 ```
 
+#### WebGL2 and WebGPU
+
+Bevy support for WebGPU is being worked on, but is currently experimental.
+
+If you don't enable the `webgl` feature, it will build for WebGPU by default, which may not work and has limited browser support.
```

Comment:
I think it's best we consider this a breaking change and add it to the migration guide.

Reply:
that's more of an open question...

Reply:
back to WebGL2 by default 👍